### PR TITLE
Update use-notifications-and-policy-tips.md

### PR DESCRIPTION
**Pull request by msbemba**
Added Conditions and Exceptions to the more information section

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4131

File: microsoft-365/compliance/use-notifications-and-policy-tips.md [hunk 1]
```diff
@@ -281,6 +281,8 @@ You can customize the text for policy tips separately from the email notificatio
 - [Overview of data loss prevention policies](data-loss-prevention-policies.md)
     
 - [Create a DLP policy from a template](create-a-dlp-policy-from-a-template.md)
+-
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/use-notifications-and-policy-tips.md]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - please remove the blank line.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**